pybamm/parameters/standard_parameters_lithium_ion.py

```diff
@@ -272,12 +271,17 @@ def U_p_dimensional(sto, T):
 sigma_n = sigma_n_dim * potential_scale / i_typ / L_x
 sigma_p = sigma_p_dim * potential_scale / i_typ / L_x
 sigma_cp = sigma_cp_dimensional * potential_scale / i_typ / L_x
-
+sigma_cn_prime = sigma_cn * delta ** 2
+sigma_n_prime = sigma_n * delta
+sigma_p_prime = sigma_p * delta
+sigma_cp_prime = sigma_cp * delta ** 2
+sigma_cn_dbl_prime = sigma_cn_prime * delta
+sigma_cp_dbl_prime = sigma_cp_prime * delta
 # should rename this to avoid confusion with Butler-Volmer
-
-alpha = 1 / (sigma_cn * (L_x / L_z) ** 2 * l_cn) + 1 / (
-    sigma_cp * (L_x / L_z) ** 2 * l_cp
+alpha = 1 / (sigma_cn * delta ** 2 * l_cn) + 1 / (
+    sigma_cp * delta ** 2 * l_cp
```

Comment:
We should still find a better variable for this that doesn't clash with the Butler-Volmer symmetry factor